cli/command/environment_test.go

```diff
@@ -0,0 +1,419 @@
+package command
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/quintilesims/layer0/api/job"
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/urfave/cli"
+)
+
+func TestEnvironmentCreate_userInputErrors(t *testing.T) {
+	base, ctrl := newTestCommand(t)
+	defer ctrl.Finish()
+
+	command := NewEnvironmentCommand(base.Command())
+
+	contexts := map[string]*cli.Context{
+		"Missing NAME arg": NewContext(t, nil, nil),
+		"OS missing":       NewContext(t, Args{"env_name"}, nil),
+		"Count negative": NewContext(t, Args{"env_name"},
+			Flags{
+				"os":        "linux",
+				"min-count": "-1",
+			}),
+	}
+
+	for name, c := range contexts {
+		err := command.create(c)
```

Comment:
`if err := ...`

Reply:
[nit] we've been switching to `t.Run` for table tests, e.g. 
```
for name, c := range contexts {
    t.Run(name, func(t *testing.T){
        if err := command.create(c); err != nil { ... }
    })
}
```

Reply:
✅